Packs/CommonScripts/Scripts/RunPollingCommand/RunPollingCommand_test.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from RunPollingCommand import prepare_arg_dict
+
+
+IDS_ARGS = [
+    # sanity
+    (
+        ('ids', ['a', 'b', 'c'], None, None),
+        {'ids': 'a,b,c'},
+    ),
+    # single ID
+    (
+        ('ids', 'a', None, None),
+        {'ids': 'a'},
+    ),
+    # numeric IDs
+    (
+        ('ids', [1, 2, 3], None, None),
+        {'ids': '1,2,3'},
+    ),
+    # extra args
+    (
+        ('ids', ['a', 'b', 'c'], u'arg1', u'value1'),
+        {'ids': 'a,b,c', 'arg1': 'value1'},
+    ),
+    # extra args
+    (
+        ('ids', ['a', 'b', 'c'], [u'arg1', u'arg2'], [u'value1', u'value2']),
+        {'ids': 'a,b,c', 'arg1': 'value1', 'arg2': 'value2'},
+    ),
+    # extra args
+    (
+        ('ids', ['a', 'b', 'c'], u'arg1, arg2,arg3', [u'value1', u'value2', u'value3']),
+        {'ids': 'a,b,c', 'arg1': 'value1', 'arg2': 'value2', 'arg3': 'value3'},
+    ),
+]
+
+
```

Comment:
can we add a few more cases for the `additional_arg_names`, `additional_arg_values`?